x/storage/keeper/keeper.go

```diff
@@ -759,8 +791,36 @@ func (k Keeper) SealObject(
 	if !found {
 		return types.ErrNoSuchObject
 	}
+	store := ctx.KVStore(k.storeKey)
 
-	if objectInfo.ObjectStatus != types.OBJECT_STATUS_CREATED {
+	prevPayloadSize := objectInfo.PayloadSize
+	prevCheckSums := objectInfo.Checksums
+
+	isUpdate := objectInfo.IsUpdating
+	if isUpdate {
+		internalBucketInfo := k.MustGetInternalBucketInfo(ctx, bucketInfo.Id)
+		err := k.UnChargeObjectStoreFee(ctx, bucketInfo, internalBucketInfo, objectInfo)
+		if err != nil {
+			return err
+		}
+		k.SetInternalBucketInfo(ctx, bucketInfo.Id, internalBucketInfo)
+		err = k.DeleteObjectFromVirtualGroup(ctx, bucketInfo, objectInfo)
```

Comment:
`DeleteObjectFromVirtualGroup` will invoke `SetInternalBucketInfo`, so do we still invoke `SetInternalBucketInfo` outside here?